### PR TITLE
Recommend different clients depending on the operating system

### PR DESCRIPTION
**Pull request by JBBgameich**
This is the long overdue upstreaming of the i.kaidan.im changes :)

### CODE REVIEW

**Comment by licaon-kter** [comment]
No Quicksy for Android? No Monal for iOS / Mac OS?

**Comment by JBBgameich** [comment]
You are free to add a follow up pull request to add them.

**Comment by ge0rg** [comment]
Thanks very much for the contribution. Besides of obviously being biased regarding yaxim as the No. 1 Android client (which I'll fix on my own), it would be great to re-use the platform mapping from https://github.com/xsf/xmpp.org/tree/master/data which is Android / iOS / Linux / macOS / Windows / Browser / Other.

As @licaon-kter wrote, Monal (https://monal.im or https://itunes.apple.com/us/app/monal-free-xmpp-chat/id317711500?mt=8) is currently the preferred iOS (and maybe macOS) client. I disagree with listing  Quicksy though as it has a different use case.

Long story short, please change the "desktop" detection to be "linux" consistent with the browser user-agent string. And if you have the right strings for macOS, I'd be happy as well :)

**Comment by JBBgameich** [comment]
Honestly I don't really know which clients are the best on the Apple platforms, so I'd be happy about recommedations :)

**Comment by ge0rg** [comment]
Now that was a bit parallel. Merged in 082d9519f2be0ed5f8ee2362698b5dca61d51081 albeit with a different Monal wording.
